archunit-junit/junit4/src/main/java/com/tngtech/archunit/junit/AnalyzeClasses.java

```diff
@@ -62,6 +62,12 @@
      */
     Class<? extends LocationProvider>[] locations() default {};
 
+    /**
+     * @return Whether to import all classes on the classpath.
```

Comment:
Maybe `scan` or `analyze` instead of `import`, to better distinguish the wording from the `ImportOptions`?

Reply:
What about `Whether to look for classes on the whole classpath`? I just checked the other Javadocs and that seems consistent to `packages()`, etc...

Reply:
Sounds good